internal/featuresupport/feature_support_test.go

```diff
@@ -196,18 +196,42 @@ var _ = Describe("V2ListFeatureSupportLevels API", func() {
 			}
 			for _, platform := range platforms {
 				p := platform
-				list := GetFeatureSupportList("dummy", nil, &p)
+				list := GetFeatureSupportList(SupportLevelFilters{
+					OpenshiftVersion:     "dummy",
+					CPUArchitecture:      nil,
+					PlatformType:         &p,
+					HighAvailabilityMode: nil,
+				})
 				Expect(len(list)).To(Equal(16))
 			}
 		})
+		It("GetFeatureSupportList 4.12 with HighAvailabilityMode", func() {
```

Comment:
```suggestion
		It("GetFeatureSupportList 4.12 with HighAvailabilityModeNone", func() {
```

Reply:
This test check the features count if `HighAvailabilityMode` is set, this test is not `HighAvailabilityModeNone` specific